watson_developer_cloud/__init__.py

```diff
@@ -27,9 +27,9 @@
 from .natural_language_understanding_v1 import NaturalLanguageUnderstandingV1
 from .personality_insights_v2 import PersonalityInsightsV2
 from .personality_insights_v3 import PersonalityInsightsV3
-from .speech_to_text_v1 import SpeechToTextV1
+from .speech_to_text_v1_adapter import SpeechToTextV1Adapter as SpeechToTextV1
```

Comment:
Maybe it will be good to move all the adapter to the bottom of this file and add some lines describing the reasons